back-end/src/routes/main.routes.js

```diff
@@ -22,4 +22,6 @@ routes.use('/admin', adminRouter);
 routes.use('/images', express.static(path.join(__dirname, '../images')));
 routes.use(handleError);
 
+routes.use('*', (_req, res) => res.status(404).json({ message: 'Not Found 404' }));
```

Comment:
```suggestion
routes.use('*', (_req, res) => res.status(StatusCodes.NOT_FOUND).json(C_ERR_NOT_FOUND));
```
Caso concorde, tem que importar tanto as mensagens de erro em `./src/middlewares/helpers/error` quanto o StatusCodes da lib `'http-status-codes`.